phoebe/backend/backends.py

```diff
@@ -45,7 +45,7 @@
 
 try:
     import ellc
-except ImportError:
+except (ImportError, OSError):
```

Comment:
is it really important to catch particular exceptions? If it fails to import, we don't really care _why_ it failed, so I'd suggest triggering on _all_ exceptions here (and elsewhere in the code).

Reply:
ok, switched to `except Exception` to catch anything